<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDataBlockMonitorDialog</class>
 <widget class="QDialog" name="CDataBlockMonitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="dataBlockNameEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="blockNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>79</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>当前数据块:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dataFormatLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>79</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>数据格式:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="monitorAscii">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>ASCII</string>
   </property>
  </widget>
  <widget class="QListWidget" name="packetlistWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>511</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearListWidgetBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportPacketBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>导出...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pauseMonitorBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeDlgBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
